spicetify.go

```diff
@@ -438,6 +430,9 @@ upgrade             Upgrade spicetify latest version
 
 -v, --version       Print version number and quit
 
+--update=check      Check for new version
+--update=upgrade    Upgrade to latest available version
```

Comment:
I thought about having a oneliner (something like `--update=<check|upgrade>` but then the flag was too long and it looked bad

So I deliberately regrouped both flags without a new empty line as it looked better IMO